form_configs/Cool_Load_Predictor.json

```diff
@@ -0,0 +1,61 @@
+{
+    "Cool Load Prediction Form": {
+      "Relative Compactness": {
+        "type": "slider",
+        "min_value": 0.5,
+        "default_value": 0.7,
+        "step": 0.01,
+        "field_name": "Relative_Compactness"
```

Comment:
Your `get_prediction()` function accepts `relative_compactness`, so change the parameter to lowercase.

This issue is with all the `field_name` so change all `field_names` to lowercase. Do this for all in `Cool_Load_Prediction.json` and `Heat_Load_Prediction.json`.